packages/commcare/test/index.js

```diff
@@ -319,6 +326,107 @@ describe('getCases', () => {
       },
     ]);
   });
+
+  it('should fetch cases with pagination', async () => {
+    const objects = [
+      { case_id: '123' },
+      { case_id: '456' },
+      { case_id: '789' },
+    ];
+    testServer
+      .intercept({
+        path: `/a/${domain}/api/v0.5/case`,
+        method: 'GET',
+        query: {
+          limit: 1,
+          offset: 0,
+        },
+      })
+      .reply(200, req => {
+        const offset = req.query.offset;
+        const next =
+          req.query.offset < objects.length - 1
+            ? `offset=${offset + 1}&limit=1`
+            : null;
+        return {
+          meta: {
+            limit: 1,
+            next: next,
+            offset: offset,
+            previous: null,
+            total_count: 3,
+          },
+          objects: [objects[offset]],
+        };
+      });
+
+    testServer
```

Comment:
You really shouldn't need to duplicate this intercept block three times. What's going on here?

This is not just me being picky: the duplication of these intercepts makes it really hard to understand the test